rfcs/rfc-ppr-expired-state.md

```diff
@@ -0,0 +1,61 @@
+- Start Date: 2021-11-09
+- Target Major Version: (EPIC or User Story TAG|Link)
+- Reference Issues: (fill in existing related issues, if any)
+- Entity Issue: (leave this empty)
+- Implementation PR: (leave this empty)
+
+# Summary
+
+Build a PPR nightly batch process that runs just after midnight and transitions all database registrations 
+that expired on the previous day to an expired state. 
+- No changes may be made to expired registrations.
+- The scope is PPR API only: regardless of the decision, no change is required to the PPR UI. 
+
+# Motivation
+Implement an efficient solution to setting a registration to an expired state. Most PPR API operations require
+examining and reporting on the registration state; an expired registration is a distinct state.
+Setting the state once and referencing a well-known value for all states is a desirable outcome.
+
+# Detailed design
+
+## Create a PostgreSQL database table
+Record the status of the job in a new log table. This step may be unnecessary if pg_cron is 
```

Comment:
pg_cron is not available on cloud-sql, so that's not a good option for us.

Reply:
Removed